### PR TITLE
ESH-11355 adopt to work with GWT 2.10

### PR DESCRIPTION
**Pull request by bogdanmglowski**

### CODE REVIEW

File: archetype/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.googlecode.gwtquery</groupId>
   <artifactId>gquery-archetype</artifactId>
-  <version>1.5-SNAPSHOT</version>
+  <version>66.5</version>
```
Comment (ewolk) [inline archetype/pom.xml]: no...

Comment (ewolk) [reply archetype/pom.xml]: 1.6

Comment (ewolk) [reply archetype/pom.xml]: 1.6.IEFIX

**Comment by bartmoc** [comment]
`<APPROVED/>`

**Review by ewolk — APPROVED** [review]

**Pull request merged**